nixos/tests/web-apps/mastodon/default.nix

```
@@ -0,0 +1,5 @@
+{ system ? builtins.currentSystem, handleTestOn }:
+{
+  mastodon-standart = handleTestOn [ "x86_64-linux" "i686-linux" "aarch64-linux" ] ./mastodon-std.nix { inherit system; };
```

```suggestion
  standard = handleTestOn [ "x86_64-linux" "i686-linux" "aarch64-linux" ] ./standard.nix { inherit system; };
```
This seems like a good solution to group the tests! Now since `mastodon` then already serves as a namespace in the attribute names and file names, we can simplify this, fix a typo and make it consistent.